server/container.go

```diff
@@ -161,8 +163,12 @@ func (s *Server) createSandboxContainer(containerID string, containerName string
 			options = "ro"
 		}
 
-		//TODO(hmeng): how to use this info? Do we need to handle relabel a FS with Selinux?
-		//selinuxRelabel := mount.GetSelinuxRelabel()
+		if mount.GetSelinuxRelabel() {
+			// Need a way in kubernetes to determine if the volume is shared or private
+			if err := label.Relabel(src, sb.mountLabel, true); err != nil && err != syscall.ENOTSUP {
+				return nil, fmt.Errorf("Relabel failed %s: %q", src, err)
```

Comment:
nit: when errors aren't being logged they should begin with lowercase Relabel -> relabel.